SPEC.md

```diff
@@ -78,7 +79,7 @@ It will then look for this executable in a list of predefined directories. Once
 - `CNI_COMMAND`: indicates the desired operation; `ADD`, `DEL` or `VERSION`.
 - `CNI_CONTAINERID`: Container ID
 - `CNI_NETNS`: Path to network namespace file
-- `CNI_IFNAME`: Interface name to set up
+- `CNI_IFNAME`: Interface name to set up; plugin must honor this interface name or return an error
```

Comment:
@tomdee Also, this change clarifies what's currently the de-facto requirement for internal plugins at least; CNI_IFNAME is required for both add/del in skel.go (which all internal plugins use) and they will respect it.  I guess the spec is somewhat ambiguous, a 3rd party plugin can name the interface whatever it wants, regardless of CNI_IFNAME, but that violates some assumptions on the part of runtimes.